### PR TITLE
Badge and button

### PR DESCRIPTION
**Pull request by tanveertkd**

### CODE REVIEW

**Review by himadri2110 — APPROVED** [review]
Looks clean, good to go!

**Review by gopalbbharadva — APPROVED** [review]
Awesome UI.

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.badge {
+    position: relative;
+}
+
+.badge > div {
+    position: absolute;
+    background-color: var(--color-highlight-orange);
+    height: 10px;
+    width: 10px;
```
Comment (mimansha-swarup) [inline components/badge/badge.css:9]: just a suggestion using relative value like rem will be better

**Review by AbhishekkGautam — APPROVED** [review]
You should create one more variant like link button using anchor tag. 
Everything else looks good.

**Review by mimansha-swarup — APPROVED** [review]
all looks good 👍

File: components/badge/badge.css (hunk 1 above)
Comment (tanveertkd) [reply components/badge/badge.css:9]: I meant to change it but it totally slipped my mind. Thanks for pointing it out!

**Pull request merged**